lib/warnings.js

```diff
@@ -27,4 +27,6 @@ warning.create('FastifyDeprecation', 'FSTDEP013', 'Direct return of "trailers" f
 
 warning.create('FastifyDeprecation', 'FSTDEP014', 'You are trying to set/access the default route. This property is deprecated. Please, use setNotFoundHandler if you want to custom a 404 handler or the wildcard (*) to match all routes.')
 
+warning.create('FastifyDeprecation', 'FSTDEP015', 'The %s schema for %s: %s is missing. This may indicate the schema is not well specified.', { unlimited: true })
```

Comment:
As stated in https://github.com/fastify/fastify/pull/4647#issuecomment-1496427763, this is not a deprecation. It should not be emitted as such.

Reply:
Change to be emitted as `FastifyWarning` now.